rules-documentation/logsource-guides/windows/service/security.md

```diff
@@ -0,0 +1,3212 @@
+  - 5450
+  - 5456
+  - 5457
+  - 5458
+  - 5459
+  - 5460
+  - 5461
+  - 5462
+  - 5463
+  - 5464
+  - 5465
+  - 5466
+  - 5467
+  - 5468
+  - 5471
+  - 5472
+  - 5473
+  - 5474
+  - 5477
+
+If you're using `gpedit.msc` or similar you can enable logging for this category by following the structure below
+
+```yml
+- Computer Configuration
+    - Windows Settings
+        - Security Settings
+            - Advanced Audit Policy Configuration
+                - System Audit Policies - Local Group Policy Object
+                    - Policy Change
+                        - Audit Filtering Platform Policy Change
+                            - Success and Failure
+```
+
+Alternatively you can enable logging via `auditpol` using the following command(s):
+
+```powershell
+# Enable Success audit Only
+auditpol /set /subcategory:{0CCE9233-69AE-11D9-BED3-505054503030}, /success:enable
+
+# Enable both Success and Failure auditing
+auditpol /set /subcategory:{0CCE9233-69AE-11D9-BED3-505054503030}, /success:enable /failure:disable
+```
+
+If you want to learn more about this sub-category. You can do so via MSDN - [Learn More](https://learn.microsoft.com/en-us/windows/security/threat-protection/auditing/audit-filtering-platform-policy-change)
+
+#### MPSSVC Rule-Level Policy Change
+
+- Subcategory GUID: `{0CCE9232-69AE-11D9-BED3-505054503030}`
+- Provider: `Microsoft Windows Security Auditing`
+- Channel: `Security`
+- Event Volume: `Medium`
+- API Mapping: [Learn More](https://github.com/jsecurity101/TelemetrySource/tree/main/Microsoft-Windows-Security-Auditing)
+- EventID(s):
+  - 4944
+  - 4945
+  - 4946
+  - 4947
+  - 4948
+  - 4949
+  - 4950
+  - 4951
+  - 4952
+  - 4953
+  - 4954
+  - 4956
+  - 4957
+  - 4958
+
+If you're using `gpedit.msc` or similar you can enable logging for this category by following the structure below
+
+```yml
+- Computer Configuration
+    - Windows Settings
+        - Security Settings
+            - Advanced Audit Policy Configuration
+                - System Audit Policies - Local Group Policy Object
+                    - Policy Change
+                        - Audit MPSSVC Rule-Level Policy Change
+                            - Success and Failure
+```
+
+Alternatively you can enable logging via `auditpol` using the following command(s):
+
+```powershell
+# Enable Success audit Only
+auditpol /set /subcategory:{0CCE9232-69AE-11D9-BED3-505054503030}, /success:enable
+
+# Enable both Success and Failure auditing
+auditpol /set /subcategory:{0CCE9232-69AE-11D9-BED3-505054503030}, /success:enable /failure:disable
+```
+
+If you want to learn more about this sub-category. You can do so via MSDN - [Learn More](https://learn.microsoft.com/en-us/windows/security/threat-protection/auditing/audit-mpssvc-rule-level-policy-change)
+
+#### Other Policy Change Events
+
+- Subcategory GUID: `{0CCE9234-69AE-11D9-BED3-505054503030}`
+- Provider: `Microsoft Windows Security Auditing`
+- Channel: `Security`
+- Event Volume: `Low`
```

Comment:
This was one of the verbose logs in the test env. Maybe the MS recommendation is not true here^^

Reply:
I'll change it to Medium to High

Reply:
I'll change it to Medium to High